be/src/exec/es/es_scan_reader.h

```diff
@@ -65,6 +66,8 @@ class ESScanReader {
     bool _is_first;
     std::string _init_scroll_url;
     std::string _next_scroll_url;
+    std::string _search_url;
```

Comment:
add comments to explian the difference of these url

Reply:
OK

Reply:
_search_url explained at the ESScanReader ctor